src/shogun/machine/gp/Inference.cpp

```diff
@@ -39,18 +39,11 @@
 #include <shogun/mathematics/Math.h>
 #include <shogun/lib/Lock.h>
 
-using namespace shogun;
+#ifdef HAVE_OPENMP
+#include <omp.h>
```

Comment:
@abhinavrai44 i dont get it.. where? if you refer to `#pragma omp...` that's not an omp API per se... it's a preprocessor directive and for that you do not need to include `omp.h`